src/Pagination/Pagination.tsx

```diff
@@ -193,14 +193,16 @@ const Container = styled(Box)`
 `
 
 const PageBox = styled.button<IPageBox>`
-  background: ${({ active }) =>
-    active ? theme.colors.custard : theme.colors.cream};
+  background: ${({ active }) => active ? theme.colors.custard : theme.colors.cream};
   cursor: pointer;
   display: flex;
+  -webkit-box-pack: center;
```

Comment:
`box-pack` looks to be deprecated and not recommended. If I understand correctly, you might be trying to vertically centre the element? 

Would `place-items`, `place-content` or `place-self` achieve the same aims?

Reply:
ah thanks for spotting, I copied these from dev-tools and didn't notice this was included, we don't need this field